src/scripts/main.js

```diff
@@ -1 +1,72 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const handleClick = () => {
+    resolve('First promise was resolved');
+    clearTimeout(timeout);
+    document.removeEventListener('click', handleClick);
+  };
+
+  document.addEventListener('click', handleClick);
+
+  const timeout = setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+    document.removeEventListener('click', handleClick);
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  const handleClick = () => {
+    resolve('Second promise was resolved');
+  };
+
+  document.addEventListener('click', handleClick);
+  document.addEventListener('contextmenu', handleClick);
+});
+
+const thirdPromise = new Promise((resolve) => {
+  const handleMouseDown = (even) => {
```

Comment:
There is a typo in the parameter name `even`. It should be `event` to correctly represent the event object passed to the handler.